### ISSUE TITLE
Some ideas

### ISSUE THREAD

**Issue by magicfa** [gigocabrera/MoneyLeash2#7]
Hi Luis

some changes I did after get the clone.
change package.json 

    "@angular/common": "2.0.0-rc.4",
    "@angular/compiler": "2.0.0-rc.4",
    "@angular/core": "2.0.0-rc.4",
    "@angular/forms": "0.2.0",
    "@angular/http": "2.0.0-rc.4",
    "@angular/platform-browser": "2.0.0-rc.4",
    "@angular/platform-browser-dynamic": "2.0.0-rc.4",

get rid of the ^ to make sure that release rc.4 will be used.

I had to delete the platform/ios folder and run ionic build ios
so that the platform and all the plugins where installed by the
build process.

Need to install ionic plugin add cordova-plugin-app-version

After all these changes app was running.

I tried to use my own firebase database and set up the rules es described.
After registration of a user (will be saved in the database) you try to automatically
create /house structure. This leads to permission problems. Maybe I got something wrong!

2(index):28 FIREBASE WARNING: Exception was thrown by user callback. Error: permission_denied at /houses/-KNt_q97POfdtH4P2eyL/memberaccounttypes: Client doesn't have permission to access the desired data. at Error (native) at G (http://localhost:8100/build/js/app.bundle.js:52162:36) at Object.G (http://localhost:8100/build/js/app.bundle.js:52185:86) at http://localhost:8100/build/js/app.bundle.js:52116:503 at Cg.g.wd (http://localhost:8100/build/js/app.bundle.js:52121:104) at qg.wd (http://localhost:8100/build/js/app.bundle.js:52111:364) at $f.gg (http://localhost:8100/build/js/app.bundle.js:52109:280) at cg (http://localhost:8100/build/js/app.bundle.js:52093:464) at WebSocket.La.onmessage (http://localhost:8100/build/js/app.bundle.js:52092:245) at WebSocket.wrapFn [as _onmessage] (http://localhost:8100/build/js/zone.js:769:30) (index):28 FIREBASE WARNING: update at /houses/-KPDSYvnJIO7_9xl-E62/housemembers/RgMXWhSWjqU7XZxRi5gHAGWcJOE3 failed: permission_denied 

Best Regards
Frank

**Comment by gigocabrera** [comment]
@magicfa I updated the README file with the added instructions and updated the Firebase Rules Gist with the most current changes. Thank you for the feedback!

As for the error you're getting, this is due to  the new schema changes I made a couple of days ago and the fact that you don't have the latest Firebase rules in place. If you git pull the latest and apply the new rules to your own FB and register a new user everything should work.
